src/lib.rs

```diff
@@ -131,3 +131,22 @@ where
 {
     RequestBuilder::new(Method::TRACE, base_url)
 }
+
+mod skip_debug {
+    use std::fmt;
+
+    #[derive(Clone)]
+    pub struct SkipDebug<T>(pub T);
```

Comment:
This is somewhat hacky IMHO but I saw few alternatives and those have significant downsides:
* Use a procedural macro like [`educe`](https://crates.io/crates/educe) but that adds an additional dependency and increases build times.
* Manually implement `fmt::Debug` for the relevant types but that seems to imply an unreasonable amount of boiler plate.